src/components/ProcessForm/ProcessForm.tsx

```diff
@@ -0,0 +1,260 @@
+import React, { FunctionComponent, useEffect, useState } from "react";
+import { Col, Form } from "react-bootstrap";
+import { getBlankProcess, IPerson, IProcess, ParentOrgs, Person, ProcessTypes, SetAsideRecommendations } from "../../api/DomainObjects";
+import { IProcessValidation, ProcessValidation } from "../../utils/ProcessValidation";
+import { InfoTooltip } from "../InfoTooltip/InfoTooltip";
+import { PeoplePicker } from "../PeoplePicker/PeoplePicker";
+import { SubmittableModal } from "../SubmittableModal/SubmittableModal";
+import "./ProcessForm.css"
+
+export interface IProcessFormProps {
+    processType: ProcessTypes,
+    showModal: boolean,
+    handleClose: () => void,
+    submit: (process: IProcess) => Promise<any>
+}
+
+export const ProcessForm: FunctionComponent<IProcessFormProps> = (props) => {
+
+    const [process, setProcess] = useState<IProcess>(getBlankProcess(props.processType));
+    const [validation, setValidation] = useState<IProcessValidation>();
+
+    useEffect(() => {
+        // Update validation whenever a field changes after a submission attempt
+        if (validation) {
+            setValidation(ProcessValidation.validateProcess(process, ["OZI", "OZJ", "OZA"]));
+        } // eslint-disable-next-line
+    }, [process]);
+
+    const getNumbersOnly = (input: string): string => {
+        return input.replaceAll(new RegExp("[^0-9]", 'g'), "");
+    }
+
+    const updateTotalContractValue = (input: string): void => {
+        let currency: string = getNumbersOnly(input);
+        if (currency.length <= 13) {
+            if (currency.length > 3) {
+                let commaIndex: number = currency.length - 3;
+                while (commaIndex > 0) {
+                    currency = currency.substring(0, commaIndex) + ',' + currency.substring(commaIndex);
+                    commaIndex -= 3;
+                }
+            }
+            setProcess({ ...process, ContractValueDollars: ('$' + currency) });
+        }
+    }
+
+    const submitForm = async () => {
+        const processValidation = ProcessValidation.validateProcess(process, ["OZI", "OZJ", "OZA"]);
```

Comment:
Same as previous comment. Should we update this to be one array passed to both functions? It'll always be the same list, right?

Reply:
I was leaving any decisions like that for when I implement it as a SP list, which will probably just be implemented as a context provider